web/wp-content/mu-plugins/pantheon/pantheon-login-form-mods.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Modify the WordPress login form for Pantheon
+ */
+
+ /**
+  * Should we proceed with adding the
+  * return to Pantheon button?
+  *
+  * Only if:
+  * We are on a Pantheon subdomain and
+  * "RETURN_TO_PANTHEON_BUTTON" is not false
+  */
+$show_return_to_pantheon_button = apply_filters( 'show_return_to_pantheon_button', ( 
```

Comment:
:no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).